en/flight_modes_rover/ackermann.md

```diff
@@ -19,23 +19,23 @@ Manual modes require stick inputs from the user to drive the vehicle.
 The sticks provide the same "high level" control effects over direction and rate of movement in all manual modes:
 
 - `Left stick up/down`: Drive the rover forwards/backwards (controlling speed)
-- `Right stick left/right`: Make a left/right turn (controlling steering angle ([Manual mode](#manual-mode)) or lateral acceleration ([Acro](#acro-mode) and [Position](#position-mode))).
+- `Right stick left/right`: Make a left/right turn (controlling steering angle ([Manual mode](#manual-mode)) or yaw rate ([Acro](#acro-mode) and [Position](#position-mode))).
```

Comment:
[Manual Controls] image above still shows lateral acceleration. Can you fix up in a follow on PR?